docs/concepts/cluster-administration/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -48,10 +48,8 @@ It's enabled by default. It can be disabled:
 
 ### Marking add-on as critical
 
-To be critical an add-on has to run in `kube-system` namespace (configurable via flag)
-and have the following annotations specified:
-
-* `scheduler.alpha.kubernetes.io/critical-pod` set to empty string
-* `scheduler.alpha.kubernetes.io/tolerations` set to `[{"key":"CriticalAddonsOnly", "operator":"Exists"}]`
```

Comment:
Sorry, I still don't understand. Can you write down what text I currently have, and what text you are suggesting to replace it with?